### PR TITLE
Feature/P22-50 Show tennis players and profile preview for them

### PR DESCRIPTION
**Pull request by nevenaatic**

### CODE REVIEW

File: tennis-scheduler-web-app/src/components/TennisPlayerInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+export const TennisPlayerInfo = (props) => {
+    const [player, setPlayers] = useState({});
+    const navigate = useNavigate()
+
+    useEffect(() => {
+        setPlayers(props);
+      }, []);
+
+    const goToProfile = () => {
+        console.log(props)
```
Comment (markoacimovic) [inline tennis-scheduler-web-app/src/components/TennisPlayerInfo.jsx]: Delete this

File: tennis-scheduler-web-app/src/components/TennisPlayerInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React, { useState, useEffect } from "react";
+import Moment from 'moment';
+import { useNavigate} from "react-router-dom";
+
+export const TennisPlayerInfo = (props) => {
```
Comment (markoacimovic) [inline tennis-scheduler-web-app/src/components/TennisPlayerInfo.jsx]: Rename props to something more intuitive

File: tennis-scheduler-web-app/src/components/Navbar.jsx [hunk 3]
```diff
@@ -64,6 +65,7 @@ export const NavbarStart = () => {
                 <>
                   <Nav className="me-auto">
                     <Link className="nav-link" to="/profile">Profile</Link>
+                    <Link className='nav-link' to="/players">Players</Link>
```
Comment (markoacimovic) [inline tennis-scheduler-web-app/src/components/Navbar.jsx]: Why here Players instead of Tennis players like in admin section?

Comment (nevenaatic) [reply tennis-scheduler-web-app/src/components/Navbar.jsx]: By accident

File: tennis-scheduler-web-app/src/components/ChangePassword.jsx [hunk 4]
```diff
@@ -1,17 +1,28 @@
...
     const [passwordMessage, setPasswordMessage] = useState("");
+    const [isFormEmpty, setIsFormEmpty] = useState(false);
 
     const changePasswordSubmit = e => {
         e.preventDefault();
         setFormErrors(validation(formValues));
+        if(Object.keys(formErrors).length === 0 && !isFormEmpty){
+            changePassword(formValues).then((response)=>{
+                localStorage.setItem("token", response.data.userTokenStateDto.accessToken);
+                console.log(localStorage.getItem('token'));
```
Comment (markoacimovic) [inline tennis-scheduler-web-app/src/components/ChangePassword.jsx]: Delete this log

File: tennis-scheduler-web-app/src/components/TennisPlayers.jsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+                    {players.map((tennisPlayer) => {
+                        return (
+                            <tr>
+                                <td>{tennisPlayer.firstName} </td>
+                                <td>{tennisPlayer.lastName} </td>
+                                <td>{tennisPlayer.email} </td>
+                                <td> {tennisPlayer.birthday ? Moment(tennisPlayer.birthday).format('MMMM Do YYYY.') : "/"}</td>
+                                <td>{tennisPlayer.gender} </td>
+                                <td> <Link to={{ pathname: "/profile/" + tennisPlayer.id }} className="profile-info-link"> See profile</Link></td>
+                            </tr>
```
Comment (markoacimovic) [inline tennis-scheduler-web-app/src/components/TennisPlayers.jsx]: Extract this into new component

File: tennis-scheduler-web-app/src/components/TennisPlayers.jsx [hunk 6]
```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import { useEffect } from "react";
```
Comment (cyberpoetry17) [inline tennis-scheduler-web-app/src/components/TennisPlayers.jsx]: Single line.

**Review by markoacimovic — APPROVED** [review]

**Review by cyberpoetry17 — APPROVED** [review]

**Pull request merged**